components/paytrace/actions/list-batches/list-batches.mjs

```diff
@@ -0,0 +1,41 @@
+import { axios } from "@pipedream/platform"
+import paytrace from '../../paytrace.app.mjs';
+
+export default {
+  name: "List Batches by Date Range",
+  description: "This method can be used to export a set of batch summary details with a provided date range.  This method will return one or more batch summary records.",
+  key: 'paytrace-list-batches',
+  version: "0.0.4",
+  type: "action",
+  props: {
+    paytrace,
+    start_date: {
```

Comment:
Try to always create properties in `camelCase` name convention as much as possible
```suggestion
    startDate: {
```